PG raise error 'ERROR: NUMERIC precision 131089 must be between 1 and 1000'

## Description

The following exception happen when data insert into postgresql 
* mode: truncate_insert
* column: the destination table type is `numeric`.


## Environment

* embulk v0.8.17
* embulk-output-postgresql: 0.7.4


## Error

```
2017-02-27 19:14:01.805 +0900: Embulk v0.8.17
2017-02-27 19:14:04.702 +0900 [INFO] (0001:transaction): Loaded plugin embulk-output-postgresql (0.7.4)
2017-02-27 19:14:04.797 +0900 [INFO] (0001:transaction): Listing local files at directory '/path/to/embulk/embulk/hishidama_pg_20170227/hoge/csv' filtering filename by prefix 'sample_'
2017-02-27 19:14:04.810 +0900 [INFO] (0001:transaction): Loading files [/path/to/embulk/embulk/hishidama_pg_20170227/hoge/csv/sample_01.csv.gz]
2017-02-27 19:14:04.912 +0900 [INFO] (0001:transaction): Using local thread executor with max_threads=8 / output tasks 4 = input tasks 1 * 4
2017-02-27 19:14:04.958 +0900 [INFO] (0001:transaction): Connecting to jdbc:postgresql://127.0.0.1:5432/embulk_test options {user=user, tcpKeepAlive=true, loginTimeout=300, socketTimeout=1800}
2017-02-27 19:14:05.110 +0900 [INFO] (0001:transaction): SQL: SET search_path TO "public"
2017-02-27 19:14:05.123 +0900 [INFO] (0001:transaction): > 0.01 seconds
2017-02-27 19:14:05.124 +0900 [INFO] (0001:transaction): Using truncate_insert mode
2017-02-27 19:14:05.170 +0900 [INFO] (0001:transaction): SQL: DROP TABLE IF EXISTS "hishidama_p_11a49a00_bl_tmp000"
2017-02-27 19:14:05.172 +0900 [INFO] (0001:transaction): > 0.00 seconds
2017-02-27 19:14:05.186 +0900 [INFO] (0001:transaction): SQL: CREATE TABLE IF NOT EXISTS "hishidama_p_11a49a00_bl_tmp000" ("id" INT8, "account" NUMERIC(131089,0), "time" TIMESTAMPTZ, "purchase" TIMESTAMPTZ, "comment" TEXT)
2017-02-27 19:14:05.228 +0900 [ERROR] (0001:transaction): Operation failed (0:22023)
org.embulk.exec.PartialExecutionException: java.lang.RuntimeException: org.postgresql.util.PSQLException: ERROR: NUMERIC precision 131089 must be between 1 and 1000
  /path/to/embulk: 83
```

## Reference

* [twitter](https://twitter.com/hishidama/status/836142828342255617) (Japanese text)

## Reproduce steps.

```
embulk example hoge
embulk guess hoge/seed.yml -o config.yml
```


```yaml
in:
  type: file
  path_prefix: /path/to/embulk/hishidama_pg_20170227/hoge/csv/sample_
  decoders:
  - {type: gzip}
  parser:
    charset: UTF-8
    newline: LF
    type: csv
    delimiter: ','
    quote: '"'
    escape: '"'
    null_string: 'NULL'
    trim_if_not_quoted: false
    skip_header_lines: 1
    allow_extra_columns: false
    allow_optional_columns: false
    columns:
    - {name: id, type: long}
    - {name: account, type: long}
    - {name: time, type: timestamp, format: '%Y-%m-%d %H:%M:%S'}
    - {name: purchase, type: timestamp, format: '%Y%m%d'}
    - {name: comment, type: string}
#out: {type: stdout}

out:
  type: postgresql
  mode: truncate_insert
  host: 127.0.0.1
  database: embulk_test
  user: username
  password: xxxxxxx
  table: hishidama_pg_20170227
```

create table.

```sql
create table hishidama_pg_20170227 (id bigint not null, account numeric not null, time timestamptz, purchase timestamptz, comment text);
```

## complete log

```
2017-02-27 19:14:01.805 +0900: Embulk v0.8.17
2017-02-27 19:14:04.702 +0900 [INFO] (0001:transaction): Loaded plugin embulk-output-postgresql (0.7.4)
2017-02-27 19:14:04.797 +0900 [INFO] (0001:transaction): Listing local files at directory '/path/to/embulk/embulk/hishidama_pg_20170227/hoge/csv' filtering filename by prefix 'sample_'
2017-02-27 19:14:04.810 +0900 [INFO] (0001:transaction): Loading files [/path/to/embulk/embulk/hishidama_pg_20170227/hoge/csv/sample_01.csv.gz]
2017-02-27 19:14:04.912 +0900 [INFO] (0001:transaction): Using local thread executor with max_threads=8 / output tasks 4 = input tasks 1 * 4
2017-02-27 19:14:04.958 +0900 [INFO] (0001:transaction): Connecting to jdbc:postgresql://127.0.0.1:5432/embulk_test options {user=user, tcpKeepAlive=true, loginTimeout=300, socketTimeout=1800}
2017-02-27 19:14:05.110 +0900 [INFO] (0001:transaction): SQL: SET search_path TO "public"
2017-02-27 19:14:05.123 +0900 [INFO] (0001:transaction): > 0.01 seconds
2017-02-27 19:14:05.124 +0900 [INFO] (0001:transaction): Using truncate_insert mode
2017-02-27 19:14:05.170 +0900 [INFO] (0001:transaction): SQL: DROP TABLE IF EXISTS "hishidama_p_11a49a00_bl_tmp000"
2017-02-27 19:14:05.172 +0900 [INFO] (0001:transaction): > 0.00 seconds
2017-02-27 19:14:05.186 +0900 [INFO] (0001:transaction): SQL: CREATE TABLE IF NOT EXISTS "hishidama_p_11a49a00_bl_tmp000" ("id" INT8, "account" NUMERIC(131089,0), "time" TIMESTAMPTZ, "purchase" TIMESTAMPTZ, "comment" TEXT)
2017-02-27 19:14:05.228 +0900 [ERROR] (0001:transaction): Operation failed (0:22023)
org.embulk.exec.PartialExecutionException: java.lang.RuntimeException: org.postgresql.util.PSQLException: ERROR: NUMERIC precision 131089 must be between 1 and 1000
  ポジション: 83
	at org.embulk.exec.BulkLoader$LoaderState.buildPartialExecuteException(org/embulk/exec/BulkLoader.java:373)
	at org.embulk.exec.BulkLoader.doRun(org/embulk/exec/BulkLoader.java:591)
	at org.embulk.exec.BulkLoader.access$000(org/embulk/exec/BulkLoader.java:33)
	at org.embulk.exec.BulkLoader$1.run(org/embulk/exec/BulkLoader.java:389)
	at org.embulk.exec.BulkLoader$1.run(org/embulk/exec/BulkLoader.java:385)
	at org.embulk.spi.Exec.doWith(org/embulk/spi/Exec.java:25)
	at org.embulk.exec.BulkLoader.run(org/embulk/exec/BulkLoader.java:385)
	at org.embulk.EmbulkEmbed.run(org/embulk/EmbulkEmbed.java:180)
	at java.lang.reflect.Method.invoke(java/lang/reflect/Method.java:498)
	at org.jruby.javasupport.JavaMethod.invokeDirectWithExceptionHandling(org/jruby/javasupport/JavaMethod.java:453)
	at org.jruby.javasupport.JavaMethod.invokeDirect(org/jruby/javasupport/JavaMethod.java:314)
	at RUBY.run(/Users/user/.embulk/bin/embulk!/embulk/runner.rb:84)
	at RUBY.run(/Users/user/.embulk/bin/embulk!/embulk/command/embulk_run.rb:307)
	at RUBY.<main>(/Users/user/.embulk/bin/embulk!/embulk/command/embulk_main.rb:2)
	at org.jruby.Ruby.runInterpreter(org/jruby/Ruby.java:850)
	at org.jruby.Ruby.loadFile(org/jruby/Ruby.java:2976)
	at org.jruby.RubyKernel.requireCommon(org/jruby/RubyKernel.java:963)
	at org.jruby.RubyKernel.require(org/jruby/RubyKernel.java:956)
	at org.jruby.RubyKernel$INVOKER$s$1$0$require19.call(org/jruby/RubyKernel$INVOKER$s$1$0$require19.gen)
	at RUBY.(root)(uri:classloader:/META-INF/jruby.home/lib/ruby/stdlib/rubygems/core_ext/kernel_require.rb:1)
	at Users.user.$_dot_embulk.bin.embulk.embulk.command.embulk_bundle.invokeOther66:require(Users/user/$_dot_embulk/bin/embulk/embulk/command/file:/Users/user/.embulk/bin/embulk!/embulk/command/embulk_bundle.rb:51)
	at Users.user.$_dot_embulk.bin.embulk.embulk.command.embulk_bundle.<main>(file:/Users/user/.embulk/bin/embulk!/embulk/command/embulk_bundle.rb:51)
	at java.lang.invoke.MethodHandle.invokeWithArguments(java/lang/invoke/MethodHandle.java:627)
	at org.jruby.Ruby.runScript(org/jruby/Ruby.java:834)
	at org.jruby.Ruby.runNormally(org/jruby/Ruby.java:749)
	at org.jruby.Ruby.runNormally(org/jruby/Ruby.java:767)
	at org.jruby.Ruby.runFromMain(org/jruby/Ruby.java:580)
	at org.jruby.Main.doRunFromMain(org/jruby/Main.java:425)
	at org.jruby.Main.internalRun(org/jruby/Main.java:313)
	at org.jruby.Main.run(org/jruby/Main.java:242)
	at org.jruby.Main.main(org/jruby/Main.java:204)
	at org.embulk.cli.Main.main(org/embulk/cli/Main.java:23)
	Suppressed: java.lang.NullPointerException
		at org.embulk.exec.BulkLoader.doCleanup(BulkLoader.java:494)
		at org.embulk.exec.BulkLoader$3.run(BulkLoader.java:425)
		at org.embulk.exec.BulkLoader$3.run(BulkLoader.java:421)
		at org.embulk.spi.Exec.doWith(Exec.java:25)
		at org.embulk.exec.BulkLoader.cleanup(BulkLoader.java:421)
		at org.embulk.EmbulkEmbed.run(EmbulkEmbed.java:184)
		at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
		at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
		at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
		at java.lang.reflect.Method.invoke(Method.java:498)
		at org.jruby.javasupport.JavaMethod.invokeDirectWithExceptionHandling(JavaMethod.java:453)
		at org.jruby.javasupport.JavaMethod.invokeDirect(JavaMethod.java:314)
		at org.jruby.java.invokers.InstanceMethodInvoker.call(InstanceMethodInvoker.java:46)
		at org.jruby.runtime.callsite.CachingCallSite.cacheAndCall(CachingCallSite.java:338)
		at org.jruby.runtime.callsite.CachingCallSite.call(CachingCallSite.java:163)
		at org.jruby.ir.interpreter.InterpreterEngine.processCall(InterpreterEngine.java:315)
		at org.jruby.ir.interpreter.StartupInterpreterEngine.interpret(StartupInterpreterEngine.java:73)
		at org.jruby.ir.interpreter.InterpreterEngine.interpret(InterpreterEngine.java:90)
		at org.jruby.internal.runtime.methods.MixedModeIRMethod.INTERPRET_METHOD(MixedModeIRMethod.java:214)
		at org.jruby.internal.runtime.methods.MixedModeIRMethod.call(MixedModeIRMethod.java:200)
		at org.jruby.internal.runtime.methods.DynamicMethod.call(DynamicMethod.java:205)
		at org.jruby.runtime.callsite.CachingCallSite.cacheAndCall(CachingCallSite.java:358)
		at org.jruby.runtime.callsite.CachingCallSite.call(CachingCallSite.java:195)
		at org.jruby.ir.interpreter.InterpreterEngine.processCall(InterpreterEngine.java:324)
		at org.jruby.ir.interpreter.StartupInterpreterEngine.interpret(StartupInterpreterEngine.java:73)
		at org.jruby.ir.interpreter.InterpreterEngine.interpret(InterpreterEngine.java:84)
		at org.jruby.internal.runtime.methods.MixedModeIRMethod.INTERPRET_METHOD(MixedModeIRMethod.java:179)
		at org.jruby.internal.runtime.methods.MixedModeIRMethod.call(MixedModeIRMethod.java:165)
		at org.jruby.internal.runtime.methods.DynamicMethod.call(DynamicMethod.java:197)
		at org.jruby.runtime.callsite.CachingCallSite.cacheAndCall(CachingCallSite.java:338)
		at org.jruby.runtime.callsite.CachingCallSite.call(CachingCallSite.java:163)
		at org.jruby.ir.interpreter.InterpreterEngine.processCall(InterpreterEngine.java:315)
		at org.jruby.ir.interpreter.StartupInterpreterEngine.interpret(StartupInterpreterEngine.java:73)
		at org.jruby.ir.interpreter.Interpreter.INTERPRET_ROOT(Interpreter.java:112)
		at org.jruby.ir.interpreter.Interpreter.execute(Interpreter.java:99)
		at org.jruby.ir.interpreter.Interpreter.execute(Interpreter.java:35)
		at org.jruby.ir.IRTranslator.execute(IRTranslator.java:42)
		at org.jruby.Ruby.runInterpreter(Ruby.java:850)
		at org.jruby.Ruby.loadFile(Ruby.java:2976)
		at org.jruby.runtime.load.LibrarySearcher$ResourceLibrary.load(LibrarySearcher.java:243)
		at org.jruby.runtime.load.LibrarySearcher$FoundLibrary.load(LibrarySearcher.java:34)
		at org.jruby.runtime.load.LoadService.tryLoadingLibraryOrScript(LoadService.java:885)
		at org.jruby.runtime.load.LoadService.smartLoadInternal(LoadService.java:525)
		at org.jruby.runtime.load.LoadService.require(LoadService.java:396)
		at org.jruby.RubyKernel.requireCommon(RubyKernel.java:963)
		at org.jruby.RubyKernel.require19(RubyKernel.java:956)
		at org.jruby.RubyKernel$INVOKER$s$1$0$require19.call(RubyKernel$INVOKER$s$1$0$require19.gen)
		at org.jruby.internal.runtime.methods.JavaMethod$JavaMethodOneOrNBlock.call(JavaMethod.java:383)
		at org.jruby.internal.runtime.methods.AliasMethod.call(AliasMethod.java:61)
		at org.jruby.runtime.callsite.CachingCallSite.call(CachingCallSite.java:161)
		at org.jruby.ir.interpreter.InterpreterEngine.processCall(InterpreterEngine.java:315)
		at org.jruby.ir.interpreter.StartupInterpreterEngine.interpret(StartupInterpreterEngine.java:73)
		at org.jruby.ir.interpreter.InterpreterEngine.interpret(InterpreterEngine.java:84)
		at org.jruby.internal.runtime.methods.MixedModeIRMethod.INTERPRET_METHOD(MixedModeIRMethod.java:179)
		at org.jruby.internal.runtime.methods.MixedModeIRMethod.call(MixedModeIRMethod.java:165)
		at org.jruby.internal.runtime.methods.DynamicMethod.call(DynamicMethod.java:197)
		at org.jruby.runtime.callsite.CachingCallSite.cacheAndCall(CachingCallSite.java:338)
		at org.jruby.runtime.callsite.CachingCallSite.call(CachingCallSite.java:163)
		at Users.user.$_dot_embulk.bin.embulk.embulk.command.embulk_bundle.invokeOther66:require(file:/Users/user/.embulk/bin/embulk!/embulk/command/embulk_bundle.rb:51)
		at Users.user.$_dot_embulk.bin.embulk.embulk.command.embulk_bundle.RUBY$script(file:/Users/user/.embulk/bin/embulk!/embulk/command/embulk_bundle.rb:51)
		at java.lang.invoke.MethodHandle.invokeWithArguments(MethodHandle.java:627)
		at org.jruby.ir.Compiler$1.load(Compiler.java:111)
		at org.jruby.Ruby.runScript(Ruby.java:834)
		at org.jruby.Ruby.runNormally(Ruby.java:749)
		at org.jruby.Ruby.runNormally(Ruby.java:767)
		at org.jruby.Ruby.runFromMain(Ruby.java:580)
		at org.jruby.Main.doRunFromMain(Main.java:425)
		at org.jruby.Main.internalRun(Main.java:313)
		at org.jruby.Main.run(Main.java:242)
		at org.jruby.Main.main(Main.java:204)
		at org.embulk.cli.Main.main(Main.java:23)
Caused by: java.lang.RuntimeException: org.postgresql.util.PSQLException: ERROR: NUMERIC precision 131089 must be between 1 and 1000
  ポジション: 83
	at org.embulk.output.jdbc.AbstractJdbcOutputPlugin.begin(AbstractJdbcOutputPlugin.java:393)
	at org.embulk.output.jdbc.AbstractJdbcOutputPlugin.transaction(AbstractJdbcOutputPlugin.java:350)
	at org.embulk.exec.BulkLoader$4$1$1.transaction(BulkLoader.java:544)
	at org.embulk.exec.LocalExecutorPlugin.transaction(LocalExecutorPlugin.java:54)
	at org.embulk.exec.BulkLoader$4$1.run(BulkLoader.java:539)
	at org.embulk.spi.util.Filters$RecursiveControl.transaction(Filters.java:96)
	at org.embulk.spi.util.Filters.transaction(Filters.java:49)
	at org.embulk.exec.BulkLoader$4.run(BulkLoader.java:533)
	at org.embulk.spi.FileInputRunner$RunnerControl$1$1.run(FileInputRunner.java:123)
	at org.embulk.standards.CsvParserPlugin.transaction(CsvParserPlugin.java:225)
	at org.embulk.spi.FileInputRunner$RunnerControl$1.run(FileInputRunner.java:117)
	at org.embulk.spi.util.Decoders$RecursiveControl.transaction(Decoders.java:77)
	at org.embulk.spi.util.Decoders$RecursiveControl$1.run(Decoders.java:73)
	at org.embulk.standards.GzipFileDecoderPlugin.transaction(GzipFileDecoderPlugin.java:30)
	at org.embulk.spi.util.Decoders$RecursiveControl.transaction(Decoders.java:68)
	at org.embulk.spi.util.Decoders.transaction(Decoders.java:33)
	at org.embulk.spi.FileInputRunner$RunnerControl.run(FileInputRunner.java:114)
	at org.embulk.standards.LocalFileInputPlugin.resume(LocalFileInputPlugin.java:79)
	at org.embulk.standards.LocalFileInputPlugin.transaction(LocalFileInputPlugin.java:69)
	at org.embulk.spi.FileInputRunner.transaction(FileInputRunner.java:65)
	at org.embulk.exec.BulkLoader.doRun(BulkLoader.java:528)
	at org.embulk.exec.BulkLoader.access$000(BulkLoader.java:33)
	at org.embulk.exec.BulkLoader$1.run(BulkLoader.java:389)
	at org.embulk.exec.BulkLoader$1.run(BulkLoader.java:385)
	at org.embulk.spi.Exec.doWith(Exec.java:25)
	at org.embulk.exec.BulkLoader.run(BulkLoader.java:385)
	at org.embulk.EmbulkEmbed.run(EmbulkEmbed.java:180)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.jruby.javasupport.JavaMethod.invokeDirectWithExceptionHandling(JavaMethod.java:453)
	at org.jruby.javasupport.JavaMethod.invokeDirect(JavaMethod.java:314)
	at org.jruby.java.invokers.InstanceMethodInvoker.call(InstanceMethodInvoker.java:46)
	at org.jruby.runtime.callsite.CachingCallSite.cacheAndCall(CachingCallSite.java:338)
	at org.jruby.runtime.callsite.CachingCallSite.call(CachingCallSite.java:163)
	at org.jruby.ir.interpreter.InterpreterEngine.processCall(InterpreterEngine.java:315)
	at org.jruby.ir.interpreter.StartupInterpreterEngine.interpret(StartupInterpreterEngine.java:73)
	at org.jruby.ir.interpreter.InterpreterEngine.interpret(InterpreterEngine.java:90)
	at org.jruby.internal.runtime.methods.MixedModeIRMethod.INTERPRET_METHOD(MixedModeIRMethod.java:214)
	at org.jruby.internal.runtime.methods.MixedModeIRMethod.call(MixedModeIRMethod.java:200)
	at org.jruby.internal.runtime.methods.DynamicMethod.call(DynamicMethod.java:205)
	at org.jruby.runtime.callsite.CachingCallSite.cacheAndCall(CachingCallSite.java:358)
	at org.jruby.runtime.callsite.CachingCallSite.call(CachingCallSite.java:195)
	at org.jruby.ir.interpreter.InterpreterEngine.processCall(InterpreterEngine.java:324)
	at org.jruby.ir.interpreter.StartupInterpreterEngine.interpret(StartupInterpreterEngine.java:73)
	at org.jruby.ir.interpreter.InterpreterEngine.interpret(InterpreterEngine.java:84)
	at org.jruby.internal.runtime.methods.MixedModeIRMethod.INTERPRET_METHOD(MixedModeIRMethod.java:179)
	at org.jruby.internal.runtime.methods.MixedModeIRMethod.call(MixedModeIRMethod.java:165)
	at org.jruby.internal.runtime.methods.DynamicMethod.call(DynamicMethod.java:197)
	at org.jruby.runtime.callsite.CachingCallSite.cacheAndCall(CachingCallSite.java:338)
	at org.jruby.runtime.callsite.CachingCallSite.call(CachingCallSite.java:163)
	at org.jruby.ir.interpreter.InterpreterEngine.processCall(InterpreterEngine.java:315)
	at org.jruby.ir.interpreter.StartupInterpreterEngine.interpret(StartupInterpreterEngine.java:73)
	at org.jruby.ir.interpreter.Interpreter.INTERPRET_ROOT(Interpreter.java:112)
	at org.jruby.ir.interpreter.Interpreter.execute(Interpreter.java:99)
	at org.jruby.ir.interpreter.Interpreter.execute(Interpreter.java:35)
	at org.jruby.ir.IRTranslator.execute(IRTranslator.java:42)
	at org.jruby.Ruby.runInterpreter(Ruby.java:850)
	at org.jruby.Ruby.loadFile(Ruby.java:2976)
	at org.jruby.runtime.load.LibrarySearcher$ResourceLibrary.load(LibrarySearcher.java:243)
	at org.jruby.runtime.load.LibrarySearcher$FoundLibrary.load(LibrarySearcher.java:34)
	at org.jruby.runtime.load.LoadService.tryLoadingLibraryOrScript(LoadService.java:885)
	at org.jruby.runtime.load.LoadService.smartLoadInternal(LoadService.java:525)
	at org.jruby.runtime.load.LoadService.require(LoadService.java:396)
	at org.jruby.RubyKernel.requireCommon(RubyKernel.java:963)
	at org.jruby.RubyKernel.require19(RubyKernel.java:956)
	at org.jruby.RubyKernel$INVOKER$s$1$0$require19.call(RubyKernel$INVOKER$s$1$0$require19.gen)
	at org.jruby.internal.runtime.methods.JavaMethod$JavaMethodOneOrNBlock.call(JavaMethod.java:383)
	at org.jruby.internal.runtime.methods.AliasMethod.call(AliasMethod.java:61)
	at org.jruby.runtime.callsite.CachingCallSite.call(CachingCallSite.java:161)
	at org.jruby.ir.interpreter.InterpreterEngine.processCall(InterpreterEngine.java:315)
	at org.jruby.ir.interpreter.StartupInterpreterEngine.interpret(StartupInterpreterEngine.java:73)
	at org.jruby.ir.interpreter.InterpreterEngine.interpret(InterpreterEngine.java:84)
	at org.jruby.internal.runtime.methods.MixedModeIRMethod.INTERPRET_METHOD(MixedModeIRMethod.java:179)
	at org.jruby.internal.runtime.methods.MixedModeIRMethod.call(MixedModeIRMethod.java:165)
	at org.jruby.internal.runtime.methods.DynamicMethod.call(DynamicMethod.java:197)
	at org.jruby.runtime.callsite.CachingCallSite.cacheAndCall(CachingCallSite.java:338)
	at org.jruby.runtime.callsite.CachingCallSite.call(CachingCallSite.java:163)
	at Users.user.$_dot_embulk.bin.embulk.embulk.command.embulk_bundle.invokeOther66:require(file:/Users/user/.embulk/bin/embulk!/embulk/command/embulk_bundle.rb:51)
	at Users.user.$_dot_embulk.bin.embulk.embulk.command.embulk_bundle.RUBY$script(file:/Users/user/.embulk/bin/embulk!/embulk/command/embulk_bundle.rb:51)
	at java.lang.invoke.MethodHandle.invokeWithArguments(MethodHandle.java:627)
	at org.jruby.ir.Compiler$1.load(Compiler.java:111)
	at org.jruby.Ruby.runScript(Ruby.java:834)
	at org.jruby.Ruby.runNormally(Ruby.java:749)
	at org.jruby.Ruby.runNormally(Ruby.java:767)
	at org.jruby.Ruby.runFromMain(Ruby.java:580)
	at org.jruby.Main.doRunFromMain(Main.java:425)
	at org.jruby.Main.internalRun(Main.java:313)
	at org.jruby.Main.run(Main.java:242)
	at org.jruby.Main.main(Main.java:204)
	at org.embulk.cli.Main.main(Main.java:23)
Caused by: org.postgresql.util.PSQLException: ERROR: NUMERIC precision 131089 must be between 1 and 1000
  ポジション: 83
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2182)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1911)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:173)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:645)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:481)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeUpdate(AbstractJdbc2Statement.java:409)
	at org.embulk.output.jdbc.JdbcOutputConnection.executeUpdate(JdbcOutputConnection.java:497)
	at org.embulk.output.jdbc.JdbcOutputConnection.createTableIfNotExists(JdbcOutputConnection.java:123)
	at org.embulk.output.jdbc.AbstractJdbcOutputPlugin.doBegin(AbstractJdbcOutputPlugin.java:513)
	at org.embulk.output.jdbc.AbstractJdbcOutputPlugin$1.run(AbstractJdbcOutputPlugin.java:386)
	at org.embulk.output.jdbc.AbstractJdbcOutputPlugin$RetryableSQLExecution.call(AbstractJdbcOutputPlugin.java:1077)
	at org.embulk.output.jdbc.AbstractJdbcOutputPlugin$RetryableSQLExecution.call(AbstractJdbcOutputPlugin.java:1065)
	at org.embulk.spi.util.RetryExecutor.run(RetryExecutor.java:100)
	at org.embulk.spi.util.RetryExecutor.runInterruptible(RetryExecutor.java:77)
	at org.embulk.output.jdbc.AbstractJdbcOutputPlugin.withRetry(AbstractJdbcOutputPlugin.java:1049)
	at org.embulk.output.jdbc.AbstractJdbcOutputPlugin.withRetry(AbstractJdbcOutputPlugin.java:1042)
	at org.embulk.output.jdbc.AbstractJdbcOutputPlugin.begin(AbstractJdbcOutputPlugin.java:381)
	... 91 more

Error: java.lang.RuntimeException: org.postgresql.util.PSQLException: ERROR: NUMERIC precision 131089 must be between 1 and 1000
  ポジション: 83
```

--- a/embulk-output-postgresql/src/main/java/org/embulk/output/postgresql/PostgreSQLOutputConnection.java
+++ b/embulk-output-postgresql/src/main/java/org/embulk/output/postgresql/PostgreSQLOutputConnection.java
@@ -253,15 +253,17 @@ public class PostgreSQLOutputConnection
                 // but cannot create column of varchar(2147483647) .
                 return "VARCHAR";
             }
+            break;
         case "NUMERIC": // only "NUMERIC" because PostgreSQL JDBC driver will return also "NUMERIC" for the type name of decimal.
             if (c.getDataLength() > MAX_NUMERIC_PRECISION) {
                 // getDataLength for numeric without precision will return 131089 .
                 // but cannot create column of numeric(131089) .
                 return "NUMERIC";
             }
-
+            break;
         default:
-            return super.buildColumnTypeName(c);
+            break;
         }
+        return super.buildColumnTypeName(c);
     }
 }

--- a/embulk-output-postgresql/src/main/java/org/embulk/output/postgresql/PostgreSQLOutputConnection.java
+++ b/embulk-output-postgresql/src/main/java/org/embulk/output/postgresql/PostgreSQLOutputConnection.java
@@ -15,6 +15,8 @@ import org.embulk.output.jdbc.JdbcSchema;
 public class PostgreSQLOutputConnection
         extends JdbcOutputConnection
 {
+    private static final int MAX_NUMERIC_PRECISION = 1000;
+
     public PostgreSQLOutputConnection(Connection connection, String schemaName, boolean autoCommit)
             throws SQLException
     {
@@ -251,6 +253,13 @@ public class PostgreSQLOutputConnection
                 // but cannot create column of varchar(2147483647) .
                 return "VARCHAR";
             }
+        case "NUMERIC": // only "NUMERIC" because PostgreSQL JDBC driver will return also "NUMERIC" for the type name of decimal.
+            if (c.getDataLength() > MAX_NUMERIC_PRECISION) {
+                // getDataLength for numeric without precision will return 131089 .
+                // but cannot create column of numeric(131089) .
+                return "NUMERIC";
+            }
+
         default:
             return super.buildColumnTypeName(c);
         }

